sycl/test/check_device_code/kernel_arguments_as.cpp

```diff
@@ -31,34 +31,21 @@
 
 using namespace sycl;
 
-template <typename Acc> struct AccWrapper { Acc accessor; };
+template <typename Acc> struct AccWrapper {
+  Acc accessor;
+};
 
-int main() {
-
-  sycl::queue queue;
-  int array[10] = {0};
-  {
-    sycl::buffer<int, 1> buf((int *)array, sycl::range<1>(10),
-                             {sycl::property::buffer::use_host_ptr()});
-    queue.submit([&](sycl::handler &cgh) {
-      auto acc = buf.get_access<sycl::access::mode::read_write>(cgh);
+SYCL_EXTERNAL void check_address_space(
```

Comment:
This test is specifically about kernel arguments, I'm not sure if rewriting it is correct.

Reply:
That makes sense, I've gone ahead and reverted the changes.